### PR TITLE
Technical _Documentation_Page

### PR DESCRIPTION
**Pull request by faridafaqiri**
### Documentation-page: I added these things in this pull request:
- I created HTML file which have include these things inside it:
    -html, meta, title, link, inside head tag.
    - main, section,  img, p, header, code, h3, and etc tag for creating a web page.
- I created css file which have include these things include it:
   - I selected some html tags as selectors.
   - selected some properties for designing the web page.
   - added @media query for creating a responsive web page.
- I setup linters configuration .
technical-dec-page

### CODE REVIEW

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @faridafaqiri 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- You should only have one header tag in your HTML file, but you have several check the inline comments for more details.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
image

✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Survey Form and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
...
+# Technical Documentation Page 🚀
+
+## Project Description 📝
+
+> This project is a documentation about PHP lunguage. by reading this documentation you can gain a general understanding of the PHP lunguage.
+I learned about how to create a responsive web page and how to write a code inside the page that show like a code.
+
+## Demo 📸
+
+Include a demo or animated GIF of your project. You can use tools like [asciinema](https://asciinema.org/) to record your terminal sessions.
```
Comment (ahzamir) [inline README.md]: - Remove the instructions I added for you for each part.

File: README.md [hunk 2]
```diff
@@ -1 +1,42 @@
...
+```
+
+## Usage 🎯
+
+after clone the repository in your local machine you can open the html file in your browser.
+to access the code of project you can open it visual studio code editor.
+
+## Author 👩‍💻
+
+Provide your name and a link to your Social Media profiles. You can also include your email address.
```
Comment (ahzamir) [inline README.md]: - Remove the instructions I added for you for each part.

**Review by ahzamir — COMMENTED** [review]
Change these in your README.md

File: README.md (hunk 2 above)
Comment (faridafaqiri) [reply README.md]: Hi @ahzamir  
thank you very much for taking the time to provide valuable feedback on my project.
Your insights were helpful and constructive, and I have made the necessary changes.

**Review by ahzamir — APPROVED** [review]
### Hi @faridafaqiri 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
Screenshot 2024-05-27 084840

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

ExcellentNiceGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Technical Documentation Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your project has a professional README.md file

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**